js/util.js

```diff
@@ -402,6 +402,82 @@
     }
 
 
+    /*
+     * Motion mini-module.
+     *
+     * Call startTrackingMotion(). The direction property will be
+     * updated.
+     *
+     * Events:
+     *
+     */
+    var motionModule  = (function () {
+        var direction = "",
+            motionModule;
+
+        motionModule = {
+            /**
+             * Starts fetching the motion, periodically, and sets the
+             * direction property of this module. 
+             */
+            startTrackingMotion: function () {
```

Comment:
When is this initially called? It needs to be called before you can use `whenDeviceTurned` and `tiltDirection` right?